app/routes/vet/cows-bvd-present-breeder.js

```diff
@@ -0,0 +1,47 @@
+const Joi = require('joi')
+const { vetVisitData: { vetBvdResult } } = require('../../session/keys')
+const session = require('../../session')
+const getYesNoInvestigateRadios = require('../helpers/yes-no-investigate-radios')
+
+const id = 'bvdRresult'
+const errorText = 'Select one option'
```

Comment:
The error message should be phrased as `Select yes if BVD was found in the herd`, more info in the [GDS design system](https://design-system.service.gov.uk/components/radios#error-messages).